website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -10,10 +10,91 @@ These components integrate SageMaker with the portability and orchestration of K
+
+```
+export SAGEMAKER_REGION=us-east-1
+export S3_BUCKET_NAME="data-bucket-${SAGEMAKER_REGION}-$RANDOM"
+
+if [[ $SAGEMAKER_REGION == "us-east-1" ]]; then
+    aws s3api create-bucket --bucket ${S3_BUCKET_NAME} --region ${SAGEMAKER_REGION}
+else
+    aws s3api create-bucket --bucket ${S3_BUCKET_NAME} --region ${SAGEMAKER_REGION} \
+    --create-bucket-configuration LocationConstraint=${SAGEMAKER_REGION}
+fi
+
+echo ${S3_BUCKET_NAME}
+```
+Note down your S3 bucket name which will be used in the samples.
+
+## SageMaker execution IAM role
+The SageMaker training job needs an IAM role to access Amazon S3 and SageMaker. Run the following commands to create a SageMaker execution IAM role that is used by SageMaker to access AWS resources:
+
+```
+export SAGEMAKER_EXECUTION_ROLE_NAME="sagemaker-execution-role-$RANDOM"
+
+TRUST="{ \"Version\": \"2012-10-17\", \"Statement\": [ { \"Effect\": \"Allow\", \"Principal\": { \"Service\": \"sagemaker.amazonaws.com\" }, \"Action\": \"sts:AssumeRole\" } ] }"
+aws iam create-role --role-name ${SAGEMAKER_EXECUTION_ROLE_NAME} --assume-role-policy-document "$TRUST"
+aws iam attach-role-policy --role-name ${SAGEMAKER_EXECUTION_ROLE_NAME} --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+aws iam attach-role-policy --role-name ${SAGEMAKER_EXECUTION_ROLE_NAME} --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess
+
+export SAGEMAKER_EXECUTION_ROLE_ARN=$(aws iam get-role --role-name ${SAGEMAKER_EXECUTION_ROLE_NAME} --output text --query 'Role.Arn')
+
+echo $SAGEMAKER_EXECUTION_ROLE_ARN
+```
+Note down the execution role ARN to use in samples.
+
+
+
+## Prepare the dataset
+
+To train a model with SageMaker, we need an S3 bucket to store the dataset and artifacts from the training process. We will use the S3 bucket you [created earlier](../README.md#s3-bucket) and simply use the dataset at `s3://sagemaker-sample-files/datasets/image/MNIST/mnist.pkl.gz`.
+
+1. Clone this repository to use the pipelines and sample scripts.
+    ```
+    git clone https://github.com/awslabs/kubeflow-manifests.git
+    cd tests/e2e
+    ```
+1. Run the following commands to install the script dependencies and upload the processed dataset to your S3 bucket:
+    ```
+    pip install -r requirements.txt
+    python3 utils/s3_for_training/sync.py ${S3_BUCKET_NAME} ${SAGEMAKER_REGION}
+    ```
+
+## Run the sample pipeline
+
+1. Port Forward to access the Kubeflow Dashboard as you usually would and go to the Pipelines Tab. As mentioned, you should be able to see the pipeline sample called - "[Tutorial] SageMaker Training". Select to run. Make sure to either create a new experiment or use an existing one. 
```

Comment:
Lets just say, open the Pipelines Tab on the Kubeflow dashboard.
nit: drop `As mentioned`